contrib/whitelist_deps.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+# disable verify unused vars, despite the fact that they are used when sourced
```

Comment:
If this is meant to be sourced it shouldn't be executable (have `#!`).

Reply:
Done, thanks.